resolwe/storage/connectors/localconnector.py

```diff
@@ -48,6 +48,39 @@ def push(self, stream, url, chunk_size=BaseStorageConnector.CHUNK_SIZE):
                 f.write(data)
                 data_remaining = len(data) == chunk_size
 
+    def multipart_push_start(self, url, size=None):
+        """Start a multipart upload.
+
+        :returns: the upload id.
+        """
+        assert size is not None, f"{self}: size of multipart upload must be specified."
+        path = self.base_path / url
+        if not path.is_file():
+            with path.open("wb") as stream:
```

Comment:
Should it fail if the file already exists?

Reply:
This means we have a file name clash or browser re-posted the upload for some reason.

The first scenario is very unlikely but could have dire consequences (some data may be lost) and upload should be aborted. I will fix the code.